JFRWebSocket.m

```diff
@@ -717,6 +725,46 @@ -(void)dealloc
 
 /////////////////////////////////////////////////////////////////////////////
 /////////////////////////////////////////////////////////////////////////////
+@interface JFRThread ()
```

Comment:
Not specifically, no reason it couldn't be move into its own. I put it in the same file since it was a fairly small class. The only advantage of keeping it in the same file is it the runloop singleton couldn't be used by other classes. Not that it would be common practice to do this, but it being scoped within the file it isn't possible for this mistake to be made.